--- boinc-5.3.22/m4/boinc_platform.m4.orig	2006-02-27 12:23:52.930330000 +0000
+++ boinc-5.3.22/m4/boinc_platform.m4	2006-02-27 12:24:30.364669500 +0000
@@ -6,7 +6,7 @@
 	      [boinc_platform=])
   AC_MSG_CHECKING([boinc platform])
   if test -z "${boinc_platform}" ; then
-    boinc_platform=`echo $target | $SED -e 's/redhat/pc/' -e 's/[[0-9]]$//' -e 's/[[0-9]]$//' -e 's/\.$//' -e 's/[[0-9]]$//' -e 's/\.$//' -e 's/[[0-9]]$//'`
+    boinc_platform=`echo $target | $SED -e 's/pld/pc/' -e 's/redhat/pc/' -e 's/[[0-9]]$//' -e 's/[[0-9]]$//' -e 's/\.$//' -e 's/[[0-9]]$//' -e 's/\.$//' -e 's/[[0-9]]$//'`
     case "${boinc_platform}" in
       sparc-sun-solaris)
     	if test "$COMPILER_MODEL_BITS" = "64" ; then
