--- boinc	2004-10-17 20:09:04.853257320 +0000
+++ boinc.new	2004-10-17 22:00:58.980554824 +0000
@@ -11,15 +11,15 @@
 # Parameters to be set before using the script
 
 # The path where BOINC state files reside
-BOINC_PATH=/usr/local/boinc
+BOINC_PATH=/usr/share/boinc
 
 # The path where BOINC client executable resides (without the client name)
 # If client 'setiathome' is in /usr/bin, you'll have BOINC_BIN_PATH=/usr/bin
 # This path may be absolute or relative to BOINC_PATH
-BOINC_BIN_PATH=.
+BOINC_BIN_PATH=/usr/bin
 
 # LOG file for BOINC, with its absolute path
-BOINC_LOG_FILE=$BOINC_PATH/boinc.log
+BOINC_LOG_FILE=/var/log/boinc.log
 
 # Options to give to the client when it starts
 # See the documentation of the BOINC client
